appveyor.yml

```diff
@@ -1,18 +1,27 @@
 ---
 version: "{build}"
 clone_depth: 5
+init:
+  - set PATH=C:\ruby%RUBY_VERSION%\bin;C:\Program Files\7-Zip;C:\Program Files\AppVeyor\BuildAgent;C:\Program Files\Git\cmd;C:\Windows\system32;C:\Program Files;C:\Windows
```

Comment:
Why is the full PATH expanded here? Is it not enough to add a few needed entries at the start and reuse `;%PATH%`.
I am a bit concerned this might be rather fragile if AppVeyor changes anything in the directory layout.

Reply:
As to why a specific path, that is done due to duplicate exe files found in a standard Appveyor image.  For instance, the MSYS2/MinGW build system (which is enabled by -rdevkit in RubyInstaller2 builds), includes several *nix/bash commands.  These commands also exist in the 'Git for Windows' bash shell, which is included in the standard Appveyor PATH.  I believe Windows 10 also has a bash shell...

It does bring about the general question of whether a windows ruby system should have various *nix/bash commands available.

Regardless, I've used that path with appveyor scripts of several extension gems.  Also, I really doubt that Appveyor will be changing any path names in the future, just as windows has not.

As an aside, I've never seen an indication that it affects mspec/spec, but I have seen ruby code that failed when git's path contained a space (default install is to 'Program Files').  I did not account for that here, as some scripts I've added a symlink to git and adjusted PATH.

Reply:
Be happy to change it back to
```
set PATH=C:\ruby%RUBY_VERSION%\bin;%PATH%
```

Reply:
Let's try your original version, we can always change back if needed.
It seems good to avoid duplicated executables in PATH and you certainly know more about Windows than I :)